### PR TITLE
277 SuperScouting: Alliance Indicator

### PR DESCRIPTION
**Pull request by Lauren618o**
https://github.com/4201VitruvianBots/ScoutingApp2023/issues/277

### CODE REVIEW

File: SuperScouting/src/App.js [hunk 1]
```diff
@@ -37,7 +54,7 @@ function csvStringify(data) {
 class App extends React.Component {
     constructor(props) {
         super(props);
-        this.state = { signedIn: false, ScouterName: "", EventName: "", selected: 'sign-in', fouls: [], matchSchedule: null, team1: options[0], team2: options[0], team3: options[0], matchNumber: null };
+        this.state = { signedIn: false, ScouterName: "", EventName: "", selected: 'sign-in', fouls: [], matchSchedule: null, team1: options[0], team2: options[0], team3: options[0], matchNumber: null, scouterColor: null };
```
Comment (JosiahFu) [inline SuperScouting/src/App.js:57]: There's already a state called `Alliance` which stores the color (0 = red, 1 = blue). I forgot to add it to the constructor but it does get set (see line 81). Therefore I don't think you need any of the code dealing with `scouterColor`, you can just use `Alliance`

File: SuperScouting/src/App.js [hunk 2]
```diff
@@ -1,8 +1,25 @@
 import './App.css';
+// import { MultiButton } from './Form';
 import { SignIn, General } from "./Pages";
 import { options } from './Form'
 import React from "react";
 
+// const AllianceColor = () => {
```
Comment (NathanDuPont) [inline SuperScouting/src/App.js:7]: We can remove this commented out logic

File: SuperScouting/src/App.js [hunk 3]
```diff
@@ -1,8 +1,25 @@
 import './App.css';
+// import { MultiButton } from './Form';
```
Comment (NathanDuPont) [inline SuperScouting/src/App.js:2]: We can remove this

**Review by JosiahFu — CHANGES_REQUESTED** [review]

File: SuperScouting/src/Form.js [hunk 4]
```diff
@@ -146,68 +146,132 @@ class ButtonInput extends React.Component {
...
+                {this.generateButtons()}
+                <input type="hidden" name={this.state.id} value={this.state.selected} />
             </div>
-        )
-
+        );
     }
 }
 
+// class MultiButton extends React.Component {
```
Comment (NathanDuPont) [inline SuperScouting/src/Form.js:211]: Was this just a separate Multibutton component? If so, we can remove this

File: SuperScouting/src/Pages.js [hunk 5]
```diff
@@ -85,15 +85,27 @@ function SignIn(props) {
...
                     {showCheck && <div class="check"></div>}
 
                     <input type="submit" className="SAVE" value="Sign In" />
-
+                    <br></br>
                 </div>
+                {/* <br></br>
+                <br></br>
+                <br></br>
+                <br></br>
```
Comment (NathanDuPont) [inline SuperScouting/src/Pages.js:100]: We can remove these `<br>` tags

File: SuperScouting/src/Form.js (hunk 4 above)
Comment (JosiahFu) [reply SuperScouting/src/Form.js:211]: Was it a backup copy of MultiButton made before the changes?

File: SuperScouting/src/Pages.js [hunk 6]
```diff
@@ -85,15 +85,27 @@ function SignIn(props) {
                     </select>
                     <br />
                     <div className="allianceSelect">
-                        <MultiButton items={[['RED', 'Red'], ['BLUE', 'Blue']]} id="Team_Alliance" />
```
[the remark this answers is not in the document]
Comment (Lauren618o) [reply SuperScouting/src/Pages.js:88]: I put it back

**Review by kyletsubaki — APPROVED** [review]

**Comment by kyletsubaki** [comment]
SuperScouting doesn't build, so I won't be merging it into master.

**Review by JosiahFu — APPROVED** [review]

**Pull request merged**